docs/pages/versions/unversioned/sdk/video.md

```diff
@@ -38,6 +38,7 @@ export default function App() {
         ref={video}
         style={styles.video}
         source={{
+          //Be careful this link is http, if your request configuration accepts only ssl, this video cannot be loaded. 
           uri: 'http://d23dyxeqlo5psv.cloudfront.net/big_buck_bunny.mp4',
```

Comment:
Hello @Cetger, thank for mentioning this problem! 👍 

It looks like the video file is also available via HTTPS,  so it should be enough to add the `s` in the URL protocol in example:

```suggestion
          uri: 'https://d23dyxeqlo5psv.cloudfront.net/big_buck_bunny.mp4',
```

Reply:
Going to apply the suggestion and merge this, since there is no response from the author.